mappings/net/minecraft/client/gl/GlHandler.mapping

```diff
@@ -263,6 +263,9 @@ CLASS none/bov net/minecraft/client/gl/GlHandler
 	METHOD f logicOp (I)V
 	METHOD f newList (II)V
 	METHOD f drawArrays (III)V
+		ARG 0 mode
+		ARG 1 first
```

Comment:
I made it match the underlying GL15 name for consistency. The name isn't great but it's clear when you look at the documentation.